framework/src/service/ServiceReferenceBase.cpp

```diff
@@ -68,9 +68,10 @@ namespace cppmicroservices
     Any
     ServiceReferenceBase::GetProperty(std::string const& key) const
     {
-        auto l = d.Load()->coreInfo->properties.Lock();
+        auto temp = d.Load();
```

Comment:
please use a more descriptive name other than `temp`/`tmp`.

Reply:
Yes. I will do that. Oversight on my part